i18n/ko.json

```
@@ -1,1962 +1,1962 @@
-    "notification_email_username_description": "Username to use when authenticating with the email server",
-    "notification_enable_email_notifications": "Enable email notifications",
-    "notification_settings": "Notification Settings",
-    "notification_settings_description": "Manage notification settings, including email",
-    "oauth_auto_launch": "Auto launch",
-    "oauth_auto_launch_description": "Start the OAuth login flow automatically upon navigating to the login page",
-    "oauth_auto_register": "Auto register",
-    "oauth_auto_register_description": "Automatically register new users after signing in with OAuth",
-    "oauth_button_text": "Button text",
-    "oauth_client_secret_description": "Required if PKCE (Proof Key for Code Exchange) is not supported by the OAuth provider",
-    "oauth_enable_description": "Login with OAuth",
-    "oauth_mobile_redirect_uri": "Mobile redirect URI",
-    "oauth_mobile_redirect_uri_override": "Mobile redirect URI override",
-    "oauth_mobile_redirect_uri_override_description": "Enable when OAuth provider does not allow a mobile URI, like '{callback}'",
+    "add_exclusion_pattern_description": "제외 규칙을 추가합니다. *, **, ? 와 같은 glob 패턴을 사용할 수 있습니다. 예: \"Raw\" 폴더 내 모든 파일 제외 → \"**/Raw/**\", .tif 파일 제외 → \"**/*.tif\", 절대 경로 제외 → \"/path/to/ignore/**\"",
+    "asset_offline_description": "이 외부 라이브러리 항목은 디스크에서 찾을 수 없어 휴지통으로 이동되었습니다. 파일이 라이브러리 내에서 이동된 경우, 타임라인에서 새 항목을 확인하세요. 복원을 위해 Immich가 아래 경로에 접근할 수 있어야 하며, 라이브러리 스캔이 필요합니다.",
+    "authentication_settings": "인증 설정",
+    "authentication_settings_description": "비밀번호, OAuth 등 인증 설정을 관리합니다.",
+    "authentication_settings_disable_all": "모든 로그인 수단을 비활성화하시겠습니까? 로그인 기능이 완전히 중단됩니다.",
+    "authentication_settings_reenable": "다시 활성화하려면 <link>서버 명령어</link>를 사용하세요.",
+    "background_task_job": "백그라운드 작업",
+    "backup_database": "데이터베이스 덤프 생성",
+    "backup_database_enable_description": "데이터베이스 덤프 기능을 활성화합니다.",
+    "backup_keep_last_amount": "보관할 이전 덤프 수",
+    "backup_settings": "데이터베이스 덤프 설정",
+    "backup_settings_description": "데이터베이스 덤프 설정을 관리합니다. 참고: 이 작업은 상태를 모니터링하지 않으며 실패 시 알림이 없습니다.",
+    "check_all": "모두 선택",
+    "cleanup": "정리",
+    "cleared_jobs": "{job} 작업 정리됨",
+    "config_set_by_file": "현재 구성은 설정 파일에 의해 지정되어 있습니다.",
+    "confirm_delete_library": "{library} 라이브러리를 삭제하시겠습니까?",
+    "confirm_delete_library_assets": "이 라이브러리를 삭제하시겠습니까? Immich에서 {count, plural, one {항목 #개} other {항목 #개}}가 삭제되며 되돌릴 수 없습니다. 파일은 디스크에 남아 있습니다.",
+    "confirm_email_below": "계속하려면 아래에 \"{email}\"을 입력하세요.",
+    "confirm_reprocess_all_faces": "모든 얼굴을 다시 처리하시겠습니까? 이 작업은 지정된 인물 정보도 초기화합니다.",
+    "confirm_user_password_reset": "{user}님의 비밀번호를 재설정하시겠습니까?",
+    "confirm_user_pin_code_reset": "{user}님의 PIN 코드를 초기화하시겠습니까?",
+    "create_job": "작업 생성",
+    "cron_expression": "Cron 표현식",
+    "cron_expression_description": "Cron 형식을 사용해 스캔 주기를 설정합니다. 예시는 <link>Crontab Guru</link>를 참고하세요.",
+    "cron_expression_presets": "Cron 사전 설정",
```

```suggestion
    "cron_expression_presets": "Cron 표현식 사전 설정",
```
Keeping the existing translation. There doesn’t seem to be a need to remove “표현식”.